src/memoryprofiler.js

```diff
@@ -85,69 +73,44 @@ var emscriptenMemoryProfiler = {
   },
 
   onMalloc: function onMalloc(ptr, size) {
+    if (!ptr) return; // ??
```

Comment:
I assume `ptr` is what malloc returns? It can be 0 if the alloc failed or if we were asked to allocate a size of 0. Can probably remove the `// ??`